website/docs/r/mysql_flexible_server.html.markdown

```diff
@@ -93,7 +93,7 @@ The following arguments are supported:
 
 ~> **NOTE:** Creating a `GeoRestore` server requires the source server with `geo_redundant_backup_enabled` enabled.
 
-~> **NOTE:** The best practise is that it has to wait greater than 10 minutes to create the `GeoRestore` server once the source server is created.
+~> **NOTE:** The best practise is that it has to wait greater than 10 minutes to create the `GeoRestore` or `Replica` server once the source server is created.
```

Comment:
@neil-yechenwei, maybe it would be better if the doc could narrow this issue down for our customers with a link to the official MS documentation? I am not sure if this is the correct doc to link to, but something along these lines I feel would be more helpful for our end users? 🙂 
 
```suggestion
~> **NOTE:** When a server is first created it may not be immediately available for `geo restore` or `replica`. It may take a few hours for the necessary metadata to be populated. Please see the [product documentation](https://learn.microsoft.com/azure/mysql/single-server/how-to-restore-server-portal#geo-restore) for more information.
```

Reply:
Thanks. Updated PR.